src/Logger.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @file
+ * Contains \Netzstrategen\CoreStandards\Logger.
+ */
+
+namespace Netzstrategen\CoreStandards;
+
+/**
+ * Class for logging cookie consent actions.
+ */
+class Logger {
+
+  /**
+   * Returns cookie consent log file path.
+   *
+   * @return string
+   */
+  public static function getLogFile(): string {
+    $log_file = apply_filters('core-standards/cookie-consent/logfile', wp_upload_dir()['path'] . '/cookie-consent.log');
```

Comment:
Major: The file can be publicly accessed, which must not be possible.  We need to move it into a subdirectory and protected that folder by putting a .htaccess file into it.  I believe the core-standards plugin already contains functions to do.

We need to ensure that the folder is group-writable, and that the .htaccess file contains `Require all denied`